packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/AnnotationEditorUtils.ts

```diff
@@ -14,23 +12,31 @@ import { flatMap, groupBy, isEmpty } from 'lodash-es'
  * This method combines the validation schema with the entity schema base properties to create a new schema that will yield
  * the expected form with desired logic.
  * @param validationSchema
- * @param entitySchemaBaseProperties
+ * @param entitySchema
  */
 export function getJsonSchemaForForm(
   validationSchema: JSONSchema7 = {},
-  entitySchemaBaseProperties: JSONSchema7['properties'] = undefined,
+  entitySchema: JSONSchema7 = {},
 ): JSONSchema7 {
+  const entitySchemaProperties =
+    getPossibleTopLevelPropertiesInObjectSchema(entitySchema)
+
   return {
     $schema: 'http://json-schema.org/draft-07/schema#',
-    // Spread the validation schema to ensure definitions are included
+    // Merge definitions
+    definitions: {
+      ...entitySchema.definitions,
+      ...validationSchema.definitions,
+    },
```

Comment:
Fix issue where `entitySchema.definitions` was dropped, causing the error in the ticket:
> Could not find a definition for #/definitions/org.sagebionetworks-repo.model.EntityRef-1.0.0.